pkg/pool-weighted/contracts/lib/CircuitBreakerLib.sol

```diff
@@ -197,22 +197,25 @@ library CircuitBreakerLib {
             return 0;
         }
 
-        // Retrieve the weight complement passed in and bptPrice computed when the circuit breaker was set.
+        // Retrieve the weight passed in and bptPrice computed when the circuit breaker was set.
         uint256 bptPrice = circuitBreakerState.decodeUint(_BPT_PRICE_OFFSET, _BPT_PRICE_WIDTH);
-        uint256 initialWeightComplement = circuitBreakerState.decodeUint(
-            _WEIGHT_COMPLEMENT_OFFSET,
-            _WEIGHT_COMPLEMENT_WIDTH
+        uint256 initialNormalizedWeight = circuitBreakerState.decodeUint(
+            _NORMALIZED_WEIGHT_OFFSET,
+            _NORMALIZED_WEIGHT_WIDTH
         );
 
+        //uint256 weightComplement = normalizedWeight.complement();
         uint256 boundRatio;
 
-        if (initialWeightComplement == weightComplement) {
+        if (initialNormalizedWeight == normalizedWeight) {
```

Comment:
See how here we need to add the 'initial' prefix to avoid confusion. We similarly shouldn't call this the 'normalized weight' anywhere.

Reply:
bptPrice and normalizedWeight are stored when it's set. There is no prefix on bptPrice, so I also removed the prefix from normalizedWeight (we were getting away from "reference", "cached", etc.)